packages/browser/jQuery/core.ts

```
@@ -0,0 +1,130 @@
+        trimmedSelector[trimmedSelector.length - 1] === '>'
+      ) {
+        // Create element
+        const tempDiv = document.createElement('div')
+        tempDiv.innerHTML = trimmedSelector
+        const elements = tempDiv.children
+
+        Array.from(elements).forEach((el, i) => {
+          this[i] = el
+        })
+        this.length = elements.length
+        return this
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)